frontend/www/js/omegaup/components/group/Scoreboards.vue

```diff
@@ -99,23 +98,16 @@ export default class GroupScoreboards extends Vue {
   alias: null | string = null;
   description: null | string = null;
 
-  onGenerateAlias(): void {
-    if (this.title === null) {
+  @Watch('title')
+  onTitleBlur(newValue: string, oldValue: string): void {
```

Comment:
```suggestion
  onTitleChanged(newValue: string, oldValue: string): void {
```

esto ya no se refiere al evento `blur` del título, así que lo mejor sería llamarlo "Changed" por consistencia con los demás `@Watch`.